web-app/js/portal/filter/DateFilter.js

```diff
@@ -123,17 +122,14 @@ Portal.filter.DateFilter = Ext.extend(Portal.filter.Filter, {
     _getDateString: function(newDate) {
 
         if (newDate) {
-            newDate.setHours(this.timezoneCorrection);
```

Comment:
Not entirely sure about just removing that. Can you tell why this was added in the first place?